Library/Homebrew/dev-cmd/bottle.rb

```diff
@@ -232,8 +232,11 @@ def self.sudo_purge
   end
 
   def self.setup_tar_and_args!(args, mtime)
+    # TODO: Refactor and move to extend/os
```

Comment:
Sorry, would really like this to be done in this PR

Reply:
Sure, I will give a try.

> If they are default: any need to add them here?

I added those for "generic OS" that CI tests against. But we can definitely remove that for real life usage.